keyboards/pisces/pisces.c

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2021 knaruo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "pisces.h"
```

Comment:
To prevent the pin handedness from keeping the system (and matrix/keyboard) awake, you want to add this: 
```suggestion
#include "pisces.h"

const matrix_row_t matrix_mask[MATRIX_ROWS] = {
//  87654321
  0b11111110,
  0b11111110,
  0b11111111,
  0b11111110,
  0b11111110,
  0b11111111
};
```

This tells it to ignore "activity" from the specific matrix position, which includes the pull up and pull downs for the handedness.

Reply:
Thanks - I added  matrix_mask[MATRIX_ROWS].
The grid direction was actually opposite:
0b11111110 --> 0b00111111.
I verified the matrix_mask on my boards and it worked.

Reply:
Awesome, glad to hear it